src/main/java/core/basesyntax/strategy/PurchaseOperation.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax.strategy;
+
+import static javax.swing.UIManager.get;
+import static javax.swing.UIManager.put;
+
+import core.basesyntax.model.FruitTransaction;
+
+public class PurchaseOperation implements OperationHandler {
+    @Override
+    public void handle(FruitTransaction transaction) {
+        int previousQuantity = (int) get(transaction.getFruit());
+        put(transaction.getFruit(), previousQuantity - transaction.getQuantity());
+    }

```

Comment:
do not forget to add corresponding validations to all of your handlers. For example, you should check if quantity is valid etc